### ISSUE TITLE
Syntax Error for let constants

### ISSUE THREAD

**Issue by Vaberer** [nicklockwood/SwiftFormat#1379]
I enabled all the rules and this code causes a syntax error. I'm attaching the version before and after running swiftformat
```
        let fontSize: CGFloat
        if UIDevice.current.userInterfaceIdiom == .phone {
            fontSize = 16
        } else {
            fontSize = 20
        }
```

```
        let fontSize: CGFloat = if UIDevice.current.userInterfaceIdiom == .phone {
            16
        } else {
            20
        }
```
What could go wrong?

![image](https://user-images.githubusercontent.com/8090436/223712744-81854b08-b6bf-4619-a08b-ae6ce2bf52d4.png)

**Comment by nicklockwood** [comment]
This code compiles OK on the latest Swift nightly. It won't work on the latest non-beta Swift version because it relies on Swift 5.8 features.

Did you set SwiftFormat --swiftversion to 5.8 by any chance?

**Comment by nicklockwood** [comment]
@noremac yes, that's true. @calda already submitted a fix for this - it'll be in the next release.

**Comment by nicklockwood** [comment]
@Vaberer @noremac fixed in 0.51.3
